### DISCUSSION TITLE
Release Discussion: v1.75.2.yr1

### DISCUSSION THREAD

**Discussion by YellowRoseCx** [YellowRoseCx/koboldcpp-rocm#67] [General]
https://github.com/YellowRoseCx/koboldcpp-rocm/releases/tag/v1.75.2.yr1-ROCm

**Comment by nnyanpasu** [comment]
works well, just like the previous versions, I run it with the arguments: KoboldCPP-v1.75.2.yr1-ROCm.exe --multiuser 0 --threads 8 --blasthreads 8 --usecublas --gpulayers 70 --contextsize 20480
